src/renderer/containers/ReplayFileStats/GameProfileHeader.tsx

```diff
@@ -182,27 +189,68 @@ const GameDetails: React.FC<{
   if (duration === null || duration === undefined) {
     duration = _.get(stats, "lastFrame");
   }
+
   const durationLength =
-    duration !== null && duration !== undefined ? convertFrameCountToDurationString(duration, "m[m] ss[s]") : "Unknown";
+    duration !== null && duration !== undefined
+      ? file.settings.gameMode == GameMode.TARGET_TEST && file.metadata
+        ? frameToGameTimer(file.metadata?.lastFrame as number, file.settings)
+        : convertFrameCountToDurationString(duration, "m[m] ss[s]")
+      : "Unknown";
+
+  const distance = _.get(stadiumStats, "distance");
+  const units = _.get(stadiumStats, "units");
+
+  const eventDisplay = {
+    label: <EventIcon />,
+    content: monthDayHourFormat(moment(startAtDisplay)) as string,
```

Comment:
```suggestion
    content: monthDayHourFormat(moment(startAtDisplay)),
```
I don't think we need to cast this type.